src/panels/templates/AutomaticCreateCluster.json

```diff
@@ -158,21 +158,10 @@
             "name": "[parameters('resourceName')]",
             "dependsOn": [],
             "properties": {
-                "enableRBAC": "[parameters('enableRBAC')]",
```

Comment:
💡 Thanks for this. might be worth to check here for `rbac` enablement here. Rest looks goo to me. gentle fyi to @hsubramanianaks for a quick test please as well. Thanks.

Reply:
It is defaulted  already in the Automatic SKU by the AKS RP to enabled and can't be disabled